services/QuillLMS/client/app/bundles/Diagnostic/libs/grading/rematching.ts

```diff
@@ -75,14 +86,14 @@ export function rematchAll(mode: string, question: Question, questionID: string,
   });
 }
 
-export function rematchOne(response: string, mode: string, question: Question, questionID: string, callback:Function) {
+export function rematchOne(response: Response, mode: string, question: Question, questionID: string, callback:Function) {
```

Comment:
This was actually just incorrect previously and `response` was never a string here, right?

Reply:
yes, that's correct!